src/redux/actions/updatePasswordActions.js

```diff
@@ -0,0 +1,37 @@
+import "@babel/polyfill";
+import axios from "../../utils/axios";
+import {
+  UPDATE_PASSWORD_INPUT_CHANGE,
+  UPDATING_PASSWORD,
+  PASSWORD_UPDATE_SUCCESS,
+  PASSWORD_UPDATE_FAILED
+} from "../actionTypes";
+
+export const handleInputChange = (field, value) => ({
+  type: UPDATE_PASSWORD_INPUT_CHANGE,
+  payload: { field, value }
+});
+
+export const handleUpdatePassword = password => async dispatch => {
+  try {
+    dispatch({
+      type: UPDATING_PASSWORD
+    });
+    const token =
+      "eyJhbGciOiJIUzI1NiJ9.bHVjLmJheW9AZ21haWwuY29t.izmRITmo5Ru923DNd0AzpC2ULtO26-nUJU2vzS7hhGc";
+    const response = await axios.put(`/users/${token}/password`, {
```

Comment:
Should this token be hardcoded in this file, or it should be retrieved when a user request a reset password, and then, the token is sent to their email?